app/src/main/java/com/sdp/movemeet/models/FirebaseObject.java

```diff
@@ -3,24 +3,18 @@
 /**
  * An object that can be found in the Firebase Realtime Database/Firestore backend
  */
-public abstract class FirebaseObject {
+public interface FirebaseObject {
```

Comment:
Yes, going back to the "interface" version 👍. Thanks again for the time taken Monday morning to speak about this and try to find a solution to be able to retrieve the documentPath of the activities!